src/lib.rs

```diff
@@ -49,7 +49,7 @@ impl core::fmt::Display for MaxRecursionReached {
     }
 }
 
-impl core::error::Error for MaxRecursionReached {}
+impl std::error::Error for MaxRecursionReached {}
```

Comment:
Did this not break no-std builds? Shouldn't this be `cfg(feature = "std")`? I guess the no-std PR didn't add CI and no one caught that in review...

Reply:
Ah nvm, the nostd PR hasn't landed yet.